diffsync/helpers.py

```diff
@@ -390,27 +390,27 @@ def sync_model(
             return (False, model)
 
         try:
-            self.logger.debug(f"Attempting model {self.action}")
-            if self.action == "create":
+            self.logger.debug(f"Attempting model {self.action.value}")
+            if self.action == DiffSyncActions.CREATE:
                 if model is not None:
                     raise ObjectNotCreated(f"Failed to create {self.model_class.get_type()} {ids} - it already exists!")
                 model = self.model_class.create(diffsync=self.dst_diffsync, ids=ids, attrs=attrs)
-            elif self.action == "update":
+            elif self.action == DiffSyncActions.UPDATE:
                 if model is None:
                     raise ObjectNotUpdated(f"Failed to update {self.model_class.get_type()} {ids} - not found!")
                 model = model.update(attrs=attrs)
-            elif self.action == "delete":
+            elif self.action == DiffSyncActions.DELETE:
                 if model is None:
                     raise ObjectNotDeleted(f"Failed to delete {self.model_class.get_type()} {ids} - not found!")
                 model = model.delete()
             else:
-                raise ObjectCrudException(f'Unknown action "{self.action}"!')
+                raise ObjectCrudException(f'Unknown action "{self.action.value}"!')
```

Comment:
I'd probably keep this as `self.action` rather than `self.action.value` just in case we get something in that isn't even a `DiffSyncActions`?

Reply:
Good point